### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by dellius-alexander** [pypi/support#3946]
### PyPI Username

dalexander2israel

### Reason for request

I recently replaced my cell phone which was damaged. I could not import the apps due to the damage to my phone was extensive. I need help to recover my account. I do not have 2FA setup on my new phone. My email address associated with the account is "dellius.alexander@gmail.com". Please help...

Sincerely,
Dellius Alexander

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by Thespi-Brain** [comment]
Hi @dellius-alexander,

I have initiated an account recovery process and an email is being sent to the primary email address associated with your account. Please follow the instructions in that email and reply to it, and we will go from there. Thanks!

**Comment by Thespi-Brain** [comment]
Hi @dellius-alexander,

I am following up regarding your request. Are you still having an issue regarding your recovery codes and logging in? Let us know! I will keep this issue open for another week unless I hear from you earlier. After that, if you are still having trouble, feel free to create a new request. Thanks!

**Comment by Thespi-Brain** [comment]
Closing due to non response.

**Comment by dellius-alexander** [comment]
Yes. I need to recover my account. This request was sent a long time ago. I
am just not getting a response. WOW....
